library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java

```diff
@@ -40,6 +43,12 @@ public static boolean isEmoji(String message) {
                 "[\u231A\u231B\u2328\u23CF\u23E9-\u23F3\u23F8-\u23FA]\uFE0F?)+");
     }
 
+    public static boolean isHasMarkdown(String message) {
+        String pattenr = "[*@_~#{\\[\\]\"'|>]";
+        boolean isHasMarkdown = Pattern.compile(pattenr).matcher(message).find();
```

Comment:
this method is going to be called *a lot*, what about we move the `Pattern.compile` to a private attr?